Mono.Cecil/ModuleDefinition.cs

```diff
@@ -13,6 +13,11 @@
 using System.IO;
 using System.Threading;
 using SR = System.Reflection;
+#if (NETSTANDARD)
+using StrongNameKeyPair=Mono.Security.Cryptography.StrongNameKeyPair;
+#else
+using StrongNameKeyPair=System.Reflection.StrongNameKeyPair;
+#endif
```

Comment:
This creates a different and incompatible API shape between the .NET Framework and .NET Standard editions of Cecil and will break if an assembly compiled with the former uses the latter at runtime (a totally possible and supported scenario).

I had started and abandoned work on a compatible design that would solve this problem. It can be found in https://github.com/teo-tsirpanis/cecil/tree/strong-name-key-pair.